src/uniprotkb/components/protein-data-views/XRefView.tsx

```diff
@@ -346,30 +344,32 @@ const XRefsGroupedByCategory = ({
     return null;
   }
   const { databaseToDatabaseInfo } = databaseInfoMaps;
-  const infoData = sortBy(databases, ({ database }) => [
+  const infoData: InfoListItem[] = sortBy(databases, ({ database }) => [
     databaseToDatabaseInfo?.[database].implicit,
     database,
-  ]).map(
-    (
-      database
-    ): {
-      title: string;
-      content: JSX.Element;
-    } => {
-      const databaseInfo = databaseToDatabaseInfo[database.database];
-      return {
-        title: databaseInfo.displayName,
-        content: (
-          <DatabaseList
-            xrefsGoupedByDatabase={database}
-            primaryAccession={primaryAccession}
-            crc64={crc64}
-            databaseToDatabaseInfo={databaseToDatabaseInfo}
-          />
-        ),
-      };
-    }
-  );
+  ]).map((database) => {
+    const databaseInfo = databaseToDatabaseInfo[database.database];
+    return {
+      title: (
+        <Link
+          to={{
+            pathname: LocationToPath[Location.DatabaseResults],
+            search: `query=(name:${databaseInfo.name})&direct`,
```

Comment:
```suggestion
            search: `query=(name:${databaseInfo.displayName})&direct`,
```
Had a look at when `displayName` is different to `name` for all databases and this occurs three times:

```
name: RCSB-PDB != displayName: RCSB PDB
name: SOURCE_MIM != displayName: SOURCE
name: SOURCE_MGI != displayName: SOURCE
```
Using `displayName` doesn't break `RCSB-PDB` (it's a 3d db so we don't link to it anyway) and fixes the `SOURCE` issue.